### DISCUSSION TITLE
Allow setting the complexity value (cost) of a field to 0

### DISCUSSION THREAD

**Discussion by sgabler** [ChilliCream/hotchocolate#6065] [Feature Request]
We will use the operation complexity (cost) as a measure for calculating the usage of a user per month, which we will then bill.

Some fields we offer in our schema are just wrappers that don't really cost us anything to calculate, and thus should not be billable.

Example:

```graphql
query EXAMPLE {
  sites(first:10) {
    nodes {
      name
      data {
        acInverterYield {
          daily {
            dataPoints {
              value
              timestamp
            }
          }
        }
      }
    }
  }
}
```

In this example, the fields `data` and `acInverterYield` are just wrappers. The actual calculation will happen once the user selects the granularity, e.g. `daily` or `monthly`.

So in the above example we would like to set the complexity of the fields `data` and `acInverterYield` to 0 by using `.Cost(0)`, but this is not possible due to the error: "The complexity cannot be below one."

![image](https://user-images.githubusercontent.com/3625477/232795137-024949d9-e5d8-4937-abd3-c9d5c12de96a.png)

Would it be possible to allow setting the complexity to 0 in the future, or do you think that this is a wrong approach here? Curious to hear some opinions 🙂

**Comment by michaelstaib** [comment]
This is now implemented
